02-express-tutorial/app.js

```diff
@@ -1,21 +1,55 @@
 const express = require('express');
 const app = express();
-const { products } = require("./data");
+const { products, people } = require("./data.js");
+const peopleRouter = require("./routes/people.js")
 
-app.get('/', (req, res) => {
+//create logger middleware
+const logger = (req,res, next) => {
+    const method = req.method
+    const url = req.url
+    const time = new Date().getFullYear()
+    console.log(method, url, time)
+    next()
+}
+//Loads html and css files to homepage
+app.use(express.static('./methods-public'))
+//create middleware for parsing json
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+
+//creating a route to people.js
+app.use("/api/v1/people", peopleRouter);
+
+app.get('/', (req, res) =>  {
     res.send('Home Page')
 })
 
-app.use(express.static("./public"))
-
 app.get('/api/v1/test', (req,res) => {
     res.json({ message: "It worked!" })
 })
 
+// app.post('/api/v1/people', (req,res) => {
+//     const { name } = req.body
+//     if (name) {
+//         people.push({ id: people.length + 1, name: req.body.name });
+//         res.status(201).json({ success: true, name: req.body.name });
+//         return console.log(name)
+//     }
+//     else{
+//         res.status(400).json({ success: false, message: "Please provide a name" })
+//         return console.log(`No name was entered`)
+//     }
+// })
+
+// app.get('/api/v1/people', (req, res) => {
+//     res.json(people)
+// })
+
```

Comment:
nit: It's usually better just to delete old code. You can always find it again in the commit history if you need it, but usually just makes the code more difficult to read